generators/app/templates/travis.yml

```diff
@@ -1,16 +1,19 @@
 sudo: false
 language: node_js
 cache:
+  yarn: true
   directories:
     - node_modules
 notifications:
   email: false
 node_js:
   - '6'
+before_script:
+  - yarn global add codecov
 script:
   - yarn start validate
 after_success:
-  - yarn start report-coverage
+  - codecov
   - yarn start release
```

Comment:
okay cool!
I'll make PR tonight and include refactoring of semantic-release for prettier-eslint, glamorous, and nps 😄